app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +28,41 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = @deploy.pull_requests
+      @pull_request_count = @pull_requests.length
```

Comment:
A few actually do need to be instance variables so that the show.html.erb can see them. Otherwise the page breaks.